.github/auto_assign.yml

```diff
@@ -0,0 +1,15 @@
+# Set to true to add reviewers to pull requests
```

Comment:
Should we call it `autoassign-reviewers-workflow.yml` (also in the other repositories)? There is some logic in how github actions are used: they are part of a workflow https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow

Reply:
Sure, sounds like a good idea 👍 
Will update other repositories.

Reply:
will update other repos. this is for assigning reviewers using our existing github app.